app/Http/Controllers/Debug/ServerAndClientInfoController.php

```diff
@@ -3,27 +3,21 @@
 namespace App\Http\Controllers\Debug;
 
 use App\Http\Controllers\Controller;
-use Illuminate\Foundation\Auth\AuthenticatesAndRegistersUsers;
 
-class ServerAndClientInfoController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | ServerAndClientInfoController
-    |--------------------------------------------------------------------------
-    |
-    | このコントローラーはクライアントとサーバの情報を扱う
-    |
-    */
+/**
+ * ServerAndClientInfoController
+ * 
+ * このコントローラーはクライアントとサーバの情報を扱う
+ */
+class ServerAndClientInfoController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition